doc/source/cluster/kubernetes/user-guides/upgrade-guide.md

```diff
@@ -9,7 +9,7 @@ For example, if the latest Ray release is 2.7.0, KubeRay CI tests the nightly Ku
 
 * KubeRay v0.6.0: Supports all Ray versions > Ray 2.0.0
 * KubeRay v1.0.0: Supports all Ray versions > Ray 2.0.0
-* KubeRay v1.1.0: Supports Ray 2.8.0 and later. Release planned with Ray 2.10.0. 
+* KubeRay v1.1.0: Supports Ray 2.8.0 and later.
```

Comment:
do you want to also add/change into 1.1.1 here?

Reply:
I may update the upgrade guide in a separate PR. I also need to add some instructions to ask users to upgrade from v1.0.0 to v1.1.1.